external/fv3fit/fv3fit/_shared/config.py

```diff
@@ -2,13 +2,39 @@
 import fsspec
 import yaml
 import os
-from typing import Optional, Union, Sequence, List
+from typing import Dict, Optional, Union, Sequence, List
 
 
 DELP = "pressure_thickness_of_atmospheric_layer"
 MODEL_CONFIG_FILENAME = "training_config.yml"
 
 
+KERAS_MODELS: Dict[str, type] = {}
```

Comment:
I think you may be able to flesh this out more:

```suggestion
KERAS_MODELS: Dict[str, Type[Estimator]] = {}
```